main.go

```diff
@@ -152,7 +149,14 @@ func main() {
 
 				// Create cache
 				cache := job.NewMemoryJobCache(db)
-				cache.Start(time.Duration(c.Int("persist-every")) * time.Second)
+
+				duration, err := time.ParseDuration(c.String("persist-every"))
+				// if duration string could not be parsed, set duration to default
+				if err != nil {
+					cache.Start(time.Duration(5 * time.Minute))
```

Comment:
This silently fails to parse what the user specified and does something else instead. Not sure what @ajvb thinks should happen here, but hiding the error doesn't seem like the right way to go.